lib/beaker/host/unix/exec.rb

```diff
@@ -280,7 +280,7 @@ def ssh_service_restart
     case self['platform']
     when /debian|ubuntu|cumulus|huaweios/
       exec(Beaker::Command.new("service ssh restart"))
-    when /(el|centos|redhat|oracle|scientific)-[7-9]|eos-7|fedora-(1[4-9]|2[0-9]|3[0-9])|archlinux-/
+    when /amazon-\d{4,}|(el|centos|redhat|oracle|scientific)-[7-9]|eos-7|fedora-(1[4-9]|2[0-9]|3[0-9])|archlinux-/
```

Comment:
Isn't Amazon Linux 2 also systemd based? And Amazon Linux 1 is EOL and not caught by anything else, so I'd keep it simple:
```suggestion
    when /amazon|(el|centos|redhat|oracle|scientific)-[7-9]|eos-7|fedora-(1[4-9]|2[0-9]|3[0-9])|archlinux-/
```

Reply:
Ya that's true, I was trying to maintain AL1 compat, but no need for that.